src/main/kotlin/com/sourcegraph/cody/agent/protocol/ProtocolTextDocument.kt

```diff
@@ -47,8 +48,9 @@ private constructor(
       return ProtocolTextDocument(uriFor(file), text, selection)
     }
 
-    fun uriFor(file: VirtualFile): String {
-      return Rfc3986UriEncoder.encode(file.url)
+    private fun uriFor(file: VirtualFile): String {
+      val uriString = FileSystems.getDefault().getPath(file.path).toUri().toString()
```

Comment:
thanks to this we have 
```
file:///c:/Users/mikolaj/blah-blah/blah
```
instead of
```
file://c:/Users/mikolaj/blah-blah/blah
```
(note additional `/` before the disk name)


That fixes the paths in the agent. This one is related directly to the issue with:
```
Caused by: java.nio.file.InvalidPathException: Illegal char <:> at index 46: c:/Users/Jay Gohil/IdeaProjects/jaysVSDemo/\\c:\Users\Jay Gohil\IdeaProjects\jaysVSDemo\BubbleSortTest.java
```
(https://github.com/sourcegraph/jetbrains/issues/1436)

Reply:
Cool! Can we removed `Rfc3986UriEncoder` then?

Reply:
it looks like we can indeed!